frontend/.env

```
@@ -1,2 +1,19 @@
 REACT_APP_BASE_URL = http://localhost:5000
 
+# Database Configuration
+MONGO_URL=mongodb+srv://mayankbisht1107:Mayank123@exccelidraw2.6o05bxy.mongodb.net/exccelidraw2?retryWrites=true&w=majority
+
+# JWT Configuration
+JWT_SECRET=123456789
+JWT_EXPIRE=7d
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_EXPIRE key should go before the JWT_SECRET key

(UnorderedKey)